src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
And, what about the Remarks section of the docs?  Should that be brought in as remarks here?  And what's the plan for other aspects of the docs, e.g. the "Thread Safety" section?